pkg/action/executor/move_util.go

```diff
@@ -165,6 +165,11 @@ func movePod(client *kclient.Clientset, tpod *api.Pod, nodeName string, retryNum
 }
 
 func createClonePod(client *kclient.Clientset, pod *api.Pod, nodeName string) (*api.Pod, error) {
+	if !util.SupportPrivilegePod(pod) {
```

Comment:
why not put this check more forward before the action, for example the [ReScheduler.preActionCheck()](https://github.com/turbonomic/kubeturbo/blob/b312eb5787cc4edb599dbc42fb1ee778d29e7c66/pkg/action/executor/rescheduler.go#L232)